src/firefox/preferences.js

```diff
@@ -1,22 +1,26 @@
 /* @flow */
 import {WebExtError} from '../errors';
 
+// Flow Types
 
-export function getPrefs(app: string = 'firefox'): Object {
-  let appPrefs = prefs[app];
-  if (!appPrefs) {
-    throw new WebExtError(`Unsupported application: ${app}`);
-  }
-  return {
-    ...prefs.common,
-    ...appPrefs,
-  };
-}
+export type FirefoxPreferences = {
+  [key: string]: bool | string | number,
+};
+
+// NOTE: 'thuderbird' prefs are not defined on purpose, it is used in the tests to
+// be sure that if flow checks are not running and web-ext is used as a library,
+// an unknown app name raises the expected exception at runtime.
+// Preferences Maps
 
+// $FLOW_IGNORE: suppress property 'thuderbird' Property not found
```

Comment:
Could we just put the `$FLOW_IGNORE` comment on the test code instead of in this module?